addons/binding/org.openhab.binding.yeelight/ESH-INF/thing/channels.xml

```diff
@@ -37,5 +38,12 @@
 			<tag>ColorTemperature</tag>
 		</tags>
 	</channel-type>
+    
+    <!-- Command Channel -->
+	<channel-type id="command">
+		<item-type>String</item-type>
+		<label>Command</label>
+		<description>Send command directly to bulb. For advanced user only.</description>
```

Comment:
"Send **a** command directly to **the device**" is probably better.

Reply:
Fixed.